### PR TITLE
charts popup

### PR DESCRIPTION
**Pull request by anitha-deriv**
Handling Charts after logging in - Dtrader page.
Related Card - https://app.clickup.com/t/20696747/WALL-3054

### CODE REVIEW

File: cypress.config.js [hunk 1]
```diff
@@ -26,14 +26,18 @@ module.exports = defineConfig({
...
+    qaBoxBaseUrl: process.env.E2E_QABOX_BASEURL,
+    mainQaBoxBaseUrl: process.env.E2E_MAIN_QABOX_BASEURL,
     mt5Login: process.env.E2E_MT5_LOGIN,
     mt5Password: process.env.E2E_MT5_PASSWORD,
     mt5BaseUrl: process.env.E2E_MT5_BASEURL,
     endpointPath: process.env.ENDPOINT_PATH,
     appstorePath: process.env.APPSTORE_PATH,
     configServer: process.env.E2E_CONFIG_SERVER,
     configAppId: process.env.E2E_CONFIG_APPID,
-    fiatOnrampAppId: process.env.E2E_FIAT_ONRAMP_APPID,
```
Comment (markodonnell-deriv) [inline cypress.config.js:36]: Should we be removing the fiatOnrampAppId environment variable?

**Review by markodonnell-deriv — CHANGES_REQUESTED** [review]

File: cypress.config.js (hunk 1 above)
Comment (anitha-deriv) [reply cypress.config.js:36]: Hi @markodonnell-deriv , This environmental variable got changed recently. Lubega was addressing that he needs it. should we retain the onrampConfigAppId variable?

**Comment by anitha-deriv** [comment]
Verified the code locally with and without popup 
1. test-app.deriv.com(Deriv app -current prod behavior - without Popup) 
2. https://new-charts1.binary.sx - the new testlink (having the popup)
Screenshot 2023-12-12 at 2 53 39 PM
Screenshot 2023-12-12 at 3 55 16 PM

File: cypress.config.js (hunk 1 above)
Comment (markodonnell-deriv) [reply cypress.config.js:36]: I think it could be could shared between your test and @lubega-deriv 's ?

Comment (markodonnell-deriv) [reply cypress.config.js:36]: As discussed, I've understood and am happy with this change now.

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

File: cypress.config.js (hunk 1 above)
Comment (anitha-deriv) [reply cypress.config.js:36]: thank you @markodonnell-deriv

**Pull request merged**